client/my-sites/patterns/components/pattern-library/index.tsx

```diff
@@ -220,6 +222,7 @@ export const PatternLibrary = ( {
 				key={ `${ category }-tracker` }
 				searchTerm={ searchTerm }
 				referrer={ referrer }
+				patternsCount={ isFetchingPatterns ? patterns.length : undefined }
```

Comment:
```suggestion
				patternsCount={ ! isFetchingPatterns ? patterns.length : undefined }
```
Looks like we have a small logical error here.

Reply:
Yeah, thanks